tests/test_formatters.py

```diff
@@ -16,10 +17,36 @@
         ({None}, '"{None}"'),
     )
 )
-def test_json_serializer_will_deal_with_datetime(input_, expected_output):
+def test_json_serialiser_will_deal_with_datetime(input_, expected_output):
 
     log_record = Mock()
     setattr(log_record, constants.TRACE_KEY, input_)
 
     assert (
         formatters.JSONFormatter().format(log_record) == expected_output)
+
+
+def test_elasticsearch_document_serialiser():
+
+    trace = {
+        constants.REQUEST_KEY: ('should', 'be', 'serialised'),
+        constants.RESPONSE_KEY: {'also': ('should', 'be', 'serialised')},
+        constants.CONTEXT_DATA_KEY: {'should': ['NOT', 'be', 'serialised']},
```

Comment:
As discussed offline, `context_data` should be serialized because it can contain arbitrary keys